index.rst

```diff
@@ -13,13 +13,93 @@
+- Remote support will be available for fault resolution on an on-call basis. 
+- Any faults which put hardware at risk will be result in closing the AuxTel early for the night.
+- The decision for closing the AuxTel early is to be made by observing staff on summit, no additional approval is required.
+
+Daytime activities
+==================
+
+Daytime Checkouts
+-----------------
+Daytime checkouts should be run once per week, preferable on Monday afternoons. 
+They should also be run following any major software updates (such as a new cycle release), or hardware changes and fault recoveries.
+They can be performed any time during the day when the system is available. 
+
+Afternoon Calibrations and Venting
+----------------------------------
+Afternoon calibrations and venting, including opening the vent gates and enabling the fan, will be performed daily by the daytime observing specialist on a best-effort basis. 
+That is to say, if daytime engineering work within AuxTel prevents us from starting calibrations and venting before 16:00, these tasks can be skipped at the discretion of the daytime observing specialist.
+
+Nighttime Activities
+====================
+
+Nighttime Operations
+--------------------
+The AuxTel should be entirely scheduler-driven at night while in survey mode. 
+A single scheduler configuration should be provided each night to prevent detailed night plans requiring active observing support.
+Once the daytime activities are complete, the telescope can be prepare for on-sky observations and the scheduler configured at any time before twilight. 
+The AuxTel should continue to be run during the night until the observing team on the summit deems it necessary to close.
+
+Fault resolution
+----------------
+Faults which interrupt observations in this mode shall be recovered when the observing specialists on the summit have time and are not occupied by responsibilities related to Simonyi Survey Telescope operations. 
+When faults occur which result only in script failures that do not result in hardware risk or CSC components going into fault state, operations should be recovered by simply playing the script queue. 
+The scheduler and standard scripts used during observations are required to be robust against such failure, and should not leave the system in an inoperable state in the case of script failure alone.
+Faults that result in CSC components going into fault state and which require intervention from the observing specialists to recover will be recovered when possible. 
+CSC faults may result in the system remaining inoperable for extended periods of time if the observing specialists are unavailable to recover. 
+In this case, the CSCs in fault should be transitioned to Standby while they await availability to resolve to prevent warning messages being broadcast. 
```

Comment:
consider shortening to, "...should be transitioned to Standby to prevent warning messages being broadcast."